ui/app/components/clients/history.js

```diff
@@ -92,7 +92,44 @@ export default class History extends Component {
   get latestUpgradeData() {
     // {id: '1.9.0', previousVersion: null, timestampInstalled: '2021-11-03T10:23:16Z'}
     // version id is 1.9.0 or earliest upgrade post 1.9.0, timestamp is RFC3339
-    return this.args.model.versionHistory[0] || null;
+    // need to return the last in the array as the data is in ascending order
+    return this.args.model.versionHistory.at(-1) || null;
+  }
+
+  get allMinorVersionsInRange() {
+    let newArray = [];
+    // ARG TODO testing unsure if the getActivityResponse is the correct one here.
```

Comment:
@hellobontempo I figured I'd throw this question at you. Is this the correct param to be checking against the queried range?

Reply:
yes - `startTime` and `endTime` are the unformatted timestamps from the API's activity response